_layouts/default.html

```diff
@@ -10,7 +10,6 @@
       {% assign author = site.data.authors[page.author] %}{% else %}{% assign author = site.owner %}
     {% endif %}
     {% include open-graph.html %}
-    <link rel="canonical" href="{{ page.url | absolute_url | replace:'index.html','' }}">
```

Comment:
The web repo adds a canonical link, so this is safe to delete as it's redundant.

Reply:
:bulb: Would you mind adding a ticket to the `Docs App` epic to ensure we migrate this when we make the docs an independent app?

Reply:
https://algorithmia.atlassian.net/browse/FRONTEND-1756